### DISCUSSION TITLE
Ability to enable access-log buffering?

### DISCUSSION THREAD

**Discussion by antoshkin** [nginx/unit#1084] [General]
I have noticed that the request processing speed drops when I enable access logging and use the fail2ban utility (the same problem was on nginx, but there it was solved by enabling log buffering).

Do you plan to add this feature in future versions of UNIT? 
Thanks.

**Comment by tippexs** [comment]
Hi @antoshkin thansk for sharing this with us. Currently this is not on the near roadmap BUT we are currently challenging the implementation we have for Logging, Tracing and Metrics. I am more than happy to take your input as a wider discussion what we actually do to make the logging experience better while working with Unit. I would like to move this into a discussion so we can add ideas about it.
